ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,239 @@
+    paddingLeft: variables.contentPadding
+  }
+});
+
+const transactionDetails: Readonly<
+  NotifiedTransaction
+> = WalletAPI.getNotifiedTransaction();
+const entityDetails: Readonly<
+  TransactionEntity
+> = WalletAPI.getTransactionEntity();
+const subjectDetails: Readonly<
+  TransactionSubject
+> = WalletAPI.getTransactionSubject();
+
+export class FirstTransactionSummaryScreen extends React.Component<
+  Props,
+  never
+> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  /**
+   * Depending on the comparison between the amount on the notice and the amount saved remotely by the lender
+   * it will be displayed a different component. If the values differ, then the user can display both the values
+   * and a brief exmplanation related to the update.
+   */
+  private isAmountUpdated() {
+    return (
+      transactionDetails.currentAmount !== transactionDetails.notifiedAmount
+    );
+  }
+
+  private getSummary() {
+    if (this.isAmountUpdated() === false) {
+      return (
+        // if the amount had been updated, it will be displayed and presented with a brief description
+        <UpdatedPaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={transactionDetails.notifiedAmount.toString()}
+          updatedAmount={transactionDetails.currentAmount.toString()}
+          expireDate={transactionDetails.expireDate.toLocaleDateString()}
+          tranche={transactionDetails.tranche}
+        />
+      );
+    } else {
+      return (
+        <PaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={transactionDetails.currentAmount.toString()}
+          expireDate={transactionDetails.expireDate.toLocaleDateString()}
+          tranche={transactionDetails.tranche}
+        />
+      );
+    }
+  }
+
+  public render(): React.ReactNode {
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true} onPress={() => this.goBack()}>
+              <Icon name="chevron-left" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("wallet.firstTransactionSummary.header")}</Text>
+          </Body>
+        </AppHeader>
+
+        <Content original={true}>
+          <Grid style={[styles.padded, WalletStyles.backContent]}>
+            <Row>
+              <Col size={5}>
+                <View spacer={true} large={true} />
+                <H3 style={WalletStyles.white}>
+                  {" "}
+                  {" " + I18n.t("wallet.firstTransactionSummary.title")}
+                </H3>
+                <H1 style={WalletStyles.white}>
+                  {transactionDetails.paymentReason}
+                </H1>
+              </Col>
+              <Col size={1}>
+                <View spacer={true} large={true} />
+                <Image
+                  source={require("../../../img/wallet/icon-avviso-pagopa.png")}
```

Comment:
Let's use the SVG here, unless this image will be added to the icon font.

By the way, @matteodesanti, how do you decide when an image should be added to the icon font and when it should be used as an SVG?

Reply:
doesn't really matter, the font is convenient to avoid specifying the path of the image - probably adding to the font makes sense when it's used in more than one place